tests/visco_plastic_diffusion_dislocation_composite_water_fugacity.prm

```diff
@@ -0,0 +1,137 @@
+# A test for composite creep in the visco_plastic material model which includes effects from
+# water fugacity. The composite creep viscosity (vis_composite) is defined as:
+#   (vis_diffusion * vis_dislocation)/(vis_diffusion + vis_dislocation)
+# Where vis_diffusion and vis_dislocation are the normal diffusion viscosity and 
+# dislocation viscosity multiplied by the concentration of water given as H ppm Si.
+# In this model, half of the box is hydrated with a water concentratin of 100 H ppm Si,
+# and the other half of the box has 0 H ppm Si. The viscosity is therefore reduced in
+# the region where there is water relative to the region that has no water.
```

Comment:
Can you include a section of comments here (or in the Material Model section), on how the flow law parameters were modified to account from water concentration.

Reply:
Can you also include in the comments the calculation of what water viscosity should be expected.